test/openAPI/features/support/oauth_token.js

```diff
@@ -0,0 +1,216 @@
+const chai = require('chai');
+const { spec } = require('pactum');
+const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
+const {
+  localhost,
+  oauthTokenEndpoint,
+  defaultExpectedResponseTime,
+  contentTypeHeader,
+  clientResponseSchema
+} = require("./helpers/helpers");
+const { jsonToBase64 } = require("./helpers/utils");
+
+chai.use(require('chai-json-schema'));
+
+const base64ToJson = (base64String) => {
```

Comment:
But maybe it could be used to decrypt id_token payload part from the response and check if it contains all the required claims. What do you think @mdulko @KarolinaKopacz

Reply:
sounds good, but it might take a while to make this solution, I think we should hard code token right now.